File: Python3/Max Consecutive Ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        maxValue = 0
        count = 0

        for n in nums:
            if n == 1:
                count = count + 1
                maxValue = max(maxValue, count)
            else:
                count = 0
        
        return maxValue

File: C#/Max Consecutive Ones.cs
public class Solution {
    public int FindMaxConsecutiveOnes(int[] nums) {
        
        int max = 0;
        int count = 0;
        
        for(int i =0;i<nums.Length;i++)
        {
            if(nums[i]==1)
            {
                count = count + 1;
            }
            else
            {
                count = 0;
            }
            
            if(count>max) max=count;
        }
        
        return max;
        
    }
}